src/mscorlib/corefx/System/Globalization/CultureData.Windows.cs

```diff
@@ -607,5 +607,26 @@ private static CultureInfo[] EnumCultures(CultureTypes types)
         {
             throw new NotImplementedException();
         }
+
+        private static string GetConsoleFallbackName(string cultureName)
+        {
+            throw new NotImplementedException();
+        }
+
+        private bool IsSupplementalCustomCulture
+        {
+            get { throw new NotImplementedException(); }
+        }
+        
+        private bool IsWin32Installed
+        {
+            get { throw new NotImplementedException(); }
+        }
+        
+        private bool IsReplacementCulture
+        {
+            get { throw new NotImplementedException(); }
```

Comment:
Is there an issue # that should be added here associated with implementing these?